src/components/highlight/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { FC, PropsWithChildren } from 'react';
+
+import { chakra, useToken } from '@chakra-ui/react';
+
+const Highlight: FC<PropsWithChildren> = ({ children }) => {
+  const [brandPrimary] = useToken('colors', ['brand.primary']);
+
+  return (
+    <chakra.span
+      backgroundImage={`url(//s2.svgbox.net/pen-brushes.svg?ic=brush-10&color=${brandPrimary.replace(
```

Comment:
This does not really match the branded highlighter, there are multiple lines with highlighting over the text.
![image](https://user-images.githubusercontent.com/3371760/226854982-ece3c112-e619-429c-a307-be91e67efc98.png)
Whereas the branded one is one line
![image](https://user-images.githubusercontent.com/3371760/226855032-eeeb939e-15a7-4572-8751-4e85394a41b4.png)

Also, I am not sure if we are allowed to shorten the highlighted line or if it needs to be at least the width of the provided element
https://scout24-ch.frontify.com/document/2#/brand-design/our-design-element/highlighter-extension